pvlib/iotools/pvgis.py

```diff
@@ -181,17 +181,21 @@ def _parse_pvgis_tmy_basic(src):
     return data, None, None, None
 
 
-def read_pvgis_tmy(filename, outputformat='csv'):
+def read_pvgis_tmy(filename, pvgis_format=None):
     """
     Read a file downloaded from PVGIS.
 
     Parameters
     ----------
     filename : str, pathlib.Path, or file-like buffer
         Name, path, or buffer of file downloaded from PVGIS.
-    outputformat : str, default 'csv'
-        Output format of PVGIS file. Must be in
-        ``['csv', 'basic', 'epw', 'json']``.
+    pvgis_format : str, default None
+        Format of PVGIS file. Equivalent to the ``outputformat`` parameter in
+        the PVGIS TMY API. If ``None`` then the file extension will be used to
+        determine the PVGIS format to parse. For PVGIS files from the API with
+        ``outputformat='basic'``, please set `pvgis_format` to ``'basic'``. If
```

Comment:
I assume that 'basic' files have no extension?  If that's correct, I have no comment.  Otherwise I'm confused by the requirement here.

Reply:
>'basic' files have no extension

that's correct